### PR TITLE
To Add to prerequisites as per customer request

### PR DESCRIPTION
**Pull request by Lana-hl**
To Add to prerequisites from this ticket convo: https://dbtcloud.zendesk.com/agent/tickets/46505

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 1]
```diff
@@ -12,9 +12,11 @@ With the dbt Semantic Layer, you'll be able to centrally define business metrics
 Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
+- After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
 
 <Snippet src="sl-considerations-banner" />
 
+Note: please have at least one model run after installing dbt metrics package and updating packages.yml
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]: hey @Lana-hl great pr and so glad i got your first contribution 😍 ! i love this callout and thank you for linking the zendesk. so true and what do you think about adding it as a note under the 'prerequisites' section? that way the user is aware that it's a must. 

I'm suggesting removing this here so it's up under the prepreqs - wdyt

```suggestion
```

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 2]
```diff
@@ -12,9 +12,11 @@ With the dbt Semantic Layer, you'll be able to centrally define business metrics
 Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
+- After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]:
```suggestion
- **Note** &mdash; After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md (hunk 1 above)
Comment (Lana-hl) [reply website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]: @mirnawong1 looks good! Thank you so much!

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ With the dbt Semantic Layer, you'll be able to centrally define business metrics
 Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
+- **Note** &mdash; After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]:
```suggestion
-  **Note &mdash;** After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 4]
```diff
@@ -12,6 +12,7 @@ With the dbt Semantic Layer, you'll be able to centrally define business metrics
 Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
+-  **Note &mdash;** After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]:
```suggestion

**Note** &mdash; After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 5]
```diff
@@ -13,6 +13,8 @@ Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
 
+**Note** &mdash; After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]:
```suggestion
```

**Review by mirnawong1 — APPROVED** [review]
hey @Lana-hl - this pr is so great! thanks for opening this up and appreciate you + the user clarifying the docs! approved this for you, feel free to merge ✨

**Comment by mirnawong1** [comment]
i'll go ahead and mergethis for @Lana-hl - feel free to let the user know your changes are live now ✨ !

**Pull request merged**